modules/apps/object/object-js-components-web/src/main/resources/META-INF/resources/components/SelectWithOption.tsx

```diff
@@ -12,23 +12,52 @@
  * details.
  */
 
-import {ClaySelect, ClaySelectWithOption} from '@clayui/form';
-import React from 'react';
+import ClayAutocomplete from '@clayui/autocomplete';
+import DropDown from '@clayui/drop-down';
+import ClayIcon from '@clayui/icon';
+import classNames from 'classnames';
+import React, {useRef, useState} from 'react';
 
 import {FieldBase} from './FieldBase';
 
+import './Select/index.scss';
```

Comment:
@Manda-vit you can keep this import together with the one above it, it will be like:
```
import './index.scss';
import {FieldBase} from '../FieldBase';
```